municipal_finance/resources.py

```diff
@@ -140,3 +146,54 @@ class Meta:
             'period_code',
             'item',
         ]
+
+
+class CapitalTypeV2Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalTypeV2
+        import_id_fields = ['code']
+
+
+class CapitalItemsV1Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalItemsV1
+        import_id_fields = ['code']
+
+
+class CapitalItemsV2Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalItemsV2
+        import_id_fields = ['code']
+
+
+class CapitalFactsV1Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalFactsV1
+        import_id_fields = [
+            'demarcation_code',
+            'period_code',
+            'item_code',
+            'function_code',
+        ]
+
+
+class CapitalFactsV2Resource(resources.ModelResource):
+    class Meta:
+        model = CapitalFactsV2
+        import_id_fields = [
+            'demarcation_code',
+            'period_code',
+            'item',
+            'function',
+            'capital_type',
+        ]
```

Comment:
why these if the data has an id field?

Reply:
@jbothma do you means we should only use `demarcation_code` and `period_code`? What `import_id_fields` does is identify the record to replace. Pretty much the same way the upsert scripts work. Perhaps knowing this might make you less nervous about having these in production code?

Reply:
for this to be safe in production, you need to replace everything with the same demarcation_code and period_code ignoring item, function and capital_type - since you can't guarantee that you will get a replacement row with the same item, function and capital type

Reply:
![Screenshot_2020-12-05_10-14-35](https://user-images.githubusercontent.com/235801/101237570-a1cce280-36e2-11eb-9e3d-4088d4d5c95f.png)

I mean your test data has an id column. Of course this is fine, just thought I'd bring it to your attention if you intended to use just classifying columns or just unique id

Reply:
Ah! I realised the other day that the items need to have to have consistent id's to ensure the tests always refer to the correct items. I'm going to include id's in the initial fixtures. That would still not make this dependable for production import, but then I've already moved these resources to the tests.

Reply:
@jbothma I'm thinking now that I'm going to stop using resources for data import over the course of the next few days, switching to using the same code that I'll be using to import in the production code (for admin imports). In that case the stability of the tests should be fine without `id`s. Are you fine with keeping it like that for now?

Reply:
@jbothma Ah those ids aren't used at all. Just did a quick export from the DB for these. I'll exclude them from the tests now.

Reply:
yeah - totally happy with using the new import functionality when that's available - nice one

Reply:
@juriejan